index.js

```diff
@@ -1,62 +1,61 @@
-const bodyParser = require('body-parser')
-const express = require('express')
-const flash = require('connect-flash')
-const path = require('path')
-const session = require('express-session')
-const KnexSessionStore = require('connect-session-knex')(session);
-
-const config = require('./config')
-const conferences = require('./lib/conferences')
-const db = require('./lib/db')
-const format = require('./lib/format')
-const createConfController = require('./controllers/createConfController')
-const landingController = require('./controllers/landingController')
-const sendValidationEmailController = require('./controllers/sendValidationEmailController')
-const statusController = require('./controllers/statusController')
-const stats = require('./lib/stats')
-const urls = require('./urls')
+const bodyParser = require("body-parser")
+const express = require("express")
+const flash = require("connect-flash")
+const path = require("path")
+const session = require("express-session")
+const KnexSessionStore = require("connect-session-knex")(session)
+
+const config = require("./config")
+const db = require("./lib/db")
+const format = require("./lib/format")
+const createConfController = require("./controllers/createConfController")
+const landingController = require("./controllers/landingController")
+const sendValidationEmailController = require("./controllers/sendValidationEmailController")
+const statusController = require("./controllers/statusController")
+const stats = require("./lib/stats")
+const urls = require("./urls")
 
 if (config.NODE_ENV === "development") {
-  console.dir({ config });
+  console.dir({ config })
 }
 
 const app = express()
 
-app.set('view engine', 'ejs')
-app.set('views', path.join(__dirname, 'views'))
-app.locals.format = format;
+app.set("view engine", "ejs")
+app.set("views", path.join(__dirname, "views"))
+app.locals.format = format
 
-app.use('/static', express.static('static'))
+app.use("/static", express.static("static"))
 // Hack for importing css from npm package
-app.use('/~', express.static(path.join(__dirname, 'node_modules')))
-app.use(bodyParser.urlencoded({ extended: false }));
+app.use("/~", express.static(path.join(__dirname, "node_modules")))
```

Comment:
ce warning "Exposure of private files" est interessant, j'ai fait une carte trello. 
(dans une PR de refacto je propose de pas ajouter ce fix en plus, sinon on peut plus suivre)

Reply:
👍

Reply:
il faudrait exposer uniquement le design système dans les faits